src/MyHealth/SymptomLogContext.tsx

```diff
@@ -0,0 +1,91 @@
+import React, {
+  createContext,
+  FunctionComponent,
+  useContext,
+  useEffect,
+  useState,
+} from "react"
+
+import { SymptomLogEntry, HealthAssessment } from "./symptoms"
+
+const fetchLogEntries = (): Promise<SymptomLogEntry[]> => {
+  return Promise.resolve([
+    {
+      id: 1,
+      symptoms: [],
+      healthAssessment: HealthAssessment.NotAtRisk,
+      date: new Date().getTime() / 1000,
```

Comment:
is there a reason we are doing seconds rather than ms here?

Reply:
Not at all, this was just some dumb data we added for displaying something purposes, happy to remove as soon as we connect the data provider.